CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Intro
+
+If there is a bugfix or feature you'd like to see implemented, the best place to start is with a GitHub issue. First check there is not an issue raised already. If the issue has not yet been raised, it is worth raising one before making any changes yourself. That way, we can let you know whether or not this is something we are already working on.
+
+As an opensource project, 51Degrees accepts changes from the community. To propose a change:
```

Comment:
I believe Open source should be 2 words: https://en.wikipedia.org/wiki/Open_source